aws/resource_aws_workspaces_directory_test.go

```diff
@@ -181,7 +206,13 @@ func testAccCheckAwsWorkspacesDirectoryDestroy(s *terraform.State) error {
 	return nil
 }
 
-func testAccCheckAwsWorkspacesDirectoryExists(n string) resource.TestCheckFunc {
+func testAccCheckAwsWorkspacesDirectoryDisappears(v *workspaces.WorkspaceDirectory) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		return workspacesDirectoryDelete(aws.StringValue(v.DirectoryId), testAccProvider.Meta().(*AWSClient).workspacesconn)
```

Comment:
It's not clear from the test helper name that it just deletes the resource under the test and doesn't check anything actually. But the framework expects `TestCheckFunc` type, so nothing better to do here 🤷‍♂